khelo/lib/ui/flow/stats/user_stat/user_stat_screen.dart

```diff
@@ -69,360 +44,184 @@ class _UserStatScreenState extends ConsumerState<UserStatScreen>
     }
 
     return ListView(
-      padding: const EdgeInsets.only(left: 16, right: 16, bottom: 50),
-      children: [
-        _battingStats(context, state),
-        _bowlingStats(context, state),
-        _fieldingStats(context, state),
-      ],
-    );
-  }
-
-  Widget _sectionTitle(BuildContext context, String title) {
-    return Column(
-      crossAxisAlignment: CrossAxisAlignment.start,
-      children: [
-        const SizedBox(height: 24),
-        Text(
-          title,
-          style: AppTextStyle.header4
-              .copyWith(color: context.colorScheme.textPrimary),
-        ),
-        const SizedBox(height: 16),
-      ],
-    );
-  }
-
-  Widget _battingStats(BuildContext context, UserStatViewState state) {
-    final totalRuns = _getTotalRunScored(state);
-
-    return Column(
-      crossAxisAlignment: CrossAxisAlignment.start,
+      padding: const EdgeInsets.all(16) + context.mediaQueryPadding,
       children: [
         _sectionTitle(
             context, context.l10n.my_stat_stats_batting_statics_title),
-        _runScoredView(context, context.l10n.my_stat_stats_run_scored_title,
-            totalRuns.toString()),
-        const SizedBox(height: 16),
-        _averageAndStrikeRateView(context, state, false),
-        const SizedBox(height: 16),
-        _ballFacedView(context, state),
+        _battingStats(context, state.userStat),
+        const SizedBox(height: 40),
+        _sectionTitle(
+            context, context.l10n.my_stat_stats_bowling_statics_title),
+        _bowlingStats(context, state.userStat),
+        const SizedBox(height: 40),
+        _sectionTitle(
+            context, context.l10n.my_stat_stats_fielding_statics_title),
+        _fieldingStats(context, state.userStat),
       ],
     );
   }
 
-  int _getTotalRunScored(UserStatViewState state) {
-    return state.ballList
-        .where((element) => element.batsman_id == state.currentUserId)
-        .fold(0, (sum, element) => sum + element.runs_scored);
-  }
+  Widget _battingStats(BuildContext context, UserStat? userStat) {
+    if (userStat == null) return Container();
```

Comment:
returning a **Sizedbox** will be much more optimized then returning a **container**, consider to use Sizedbox.